test/AWS.Deploy.CLI.IntegrationTests/ServerModeTests.cs

```diff
@@ -62,6 +64,39 @@ public Task<AWSCredentials> ResolveCredentials()
             return Task.FromResult<AWSCredentials>(testCredentials);
         }
 
+        /// <summary>
+        /// ServerMode must only be connectable from 127.0.0.1 or localhost. This test confirms that connect attempts using
+        /// the host name fail.
+        /// </summary>
+        /// <returns></returns>
+        [Fact]
+        public async Task ConfirmLocalhostOnly()
+        {
+            var portNumber = 4900;
+            var serverCommand = new ServerModeCommand(_serviceProvider.GetRequiredService<IToolInteractiveService>(), portNumber, null, true);
+            var cancelSource = new CancellationTokenSource();
+
+            var serverTask = serverCommand.ExecuteAsync(cancelSource.Token);
```

Comment:
I don't see serverTask used anywhere below, was this intentional or did it's usage get missed?

Reply:
You are correct I was never using the `serverTask` variable nor do I need to. The lifecycle of the `serverCommand` is controlled by the cancellation token. I removed the variable declaration for `serverTask`.